### PR TITLE
Update shared components in HelpPanel

### PR DESCRIPTION
**Pull request by lyzadanger**
This PR updates shared components and icons in the `HelpPanel` component.

User-visible changes: None

Part of #4860

### CODE REVIEW

File: src/sidebar/components/HelpPanel.tsx [hunk 1]
```diff
@@ -44,13 +50,15 @@ type HelpPanelTabProps = {
...
-        target="_blank"
-      >
-        <span>{linkText}</span> <Icon name="external" classes="w-3 h-3" />
+    <div
+      // Set this element's flex-basis per its flex parent and also establish
+      // a flex container (centered on both axes)
+      className="flex-1 flex items-center justify-center border-r last-of-type:border-r-0 text-lg font-medium"
+    >
+      <Link color="text-light" href={url} target="_blank">
+        <div className="flex items-center gap-x-2">
```
Comment (lyzadanger) [inline src/sidebar/components/HelpPanel.tsx:59]: Technically, a `Link` will provide us a `flex items-center` layout, so would could have achieved the same effect (_sans_ `div`) by setting `classes="gap-x-2"` on `Link`. However, I feel that that splits the styling context awkwardly. The extra layout `div` here ensures that future devs can see the intent.

I'll continue thinking on this and similar use cases as they do come up frequently.

File: src/sidebar/components/HelpPanel.tsx [hunk 2]
```diff
@@ -44,13 +50,15 @@ type HelpPanelTabProps = {
...
   return (
-    <div className="flex-1 border-r last-of-type:border-r-0">
-      <Link
-        href={url}
-        classes="flex items-center justify-center space-x-2 text-color-text-light text-lg font-medium"
-        target="_blank"
-      >
-        <span>{linkText}</span> <Icon name="external" classes="w-3 h-3" />
+    <div
+      // Set this element's flex-basis per its flex parent and also establish
```
Comment (robertknight) [inline src/sidebar/components/HelpPanel.tsx:54]: I'm not sure what is meant by "per its flex parent" here. `flex-1` sets the `flex-basis` to `0%`, or per the Tailwind docs "Use flex-1 to allow a flex item to grow and shrink as needed, ignoring its initial size".

**Review by robertknight — APPROVED** [review]
Visually these links look fine to me. I had a query about one of the comments.

File: src/sidebar/components/HelpPanel.tsx (hunk 2 above)
Comment (lyzadanger) [reply src/sidebar/components/HelpPanel.tsx:54]: Ech, poorly-written comment on my end. I'll adjust.

**Pull request merged**